src/Components/Web/src/Forms/InputRadioContext.cs

```diff
@@ -8,48 +8,22 @@ namespace Microsoft.AspNetCore.Components.Forms;
 /// </summary>
 internal class InputRadioContext
```

Comment:
I've made this internal type more mutable so that we can retain and update existing instances instead of instantiating a new one on every render.

Reply:
```suggestion
internal sealed class InputRadioContext
```

since it helps the runtime be cleverer with callvirts and such. We need an analyzer for this